Tests/SonarScanner.MSBuild.PreProcessor.Test/AnalysisConfigProcessing/Processors/TruststorePropertiesProcessorTests.cs

```diff
@@ -27,13 +27,15 @@ namespace SonarScanner.MSBuild.PreProcessor.Test.AnalysisConfigProcessing.Proces
 public class TruststorePropertiesProcessorTests
 {
     [TestMethod]
-    public void Update_TrustStorePropertiesNullValue_Mapped()
+    public void Update_TrustStorePropertiesNullValue_NotMapped_Linux()
```

Comment:
You were right. This test case was not really testing the mapping of null values.